lib/taskjuggler/TjpSyntaxRules.rb

```diff
@@ -3267,6 +3267,23 @@ def rule_projectBodyAttributes
     arg(1, 'date', 'Alternative date to be used as current date for all ' +
         'computations')
 
+    pattern(%w( _ref !date ), lambda {
+      @project['ref'] = @val[1]
+      @scanner.addMacro(TextParser::Macro.new('ref', @val[1].to_s,
+                                              @sourceFileInfo[0]))
+      @scanner.addMacro(TextParser::Macro.new(
+        'today', @val[1].to_s(@project['timeFormat']), @sourceFileInfo[0]))
+    })
+    doc('ref', <<'EOT'
+Specify the reference date that TaskJuggler uses for calculation as date that can be
+specified and set by the user. It can be used as additional point in time to help with
+tracking tasks. If no value is specified, the current value of the system clock is used. 
+If this parameter is specified, additional black line will be drawn on a chart.
+EOT
+       )
```

Comment:
Please add a syntax test case to the test/TestSuite/Syntax/Correct/ directory and add it as example here.